modules/core/02-client/migrations/v7/expected_keepers.go

```diff
@@ -10,6 +11,6 @@ import (
 type ClientKeeper interface {
 	GetClientState(ctx sdk.Context, clientID string) (exported.ClientState, bool)
 	SetClientState(ctx sdk.Context, clientID string, clientState exported.ClientState)
-	ClientStore(ctx sdk.Context, clientID string) sdk.KVStore
+	ClientStore(ctx sdk.Context, clientID string) storetypes.KVStore
```

Comment:
Is it possible to make the change from `sdk.KVStore` to `storetypes.KVStore` in a separate pr to main? Should reduce diffs a lot?

Reply:
Yeah absolutely

Reply:
this one is next

Reply:
oops actually the answer to this one is no, I think. 

they refer to different things.

Reply:
Could you elaborate on more on the issue?

Reply:
Since 95% of sdk.KVStore things are an alias moving it all to store types in a separate pr then this pr only changes the import could work.